plugins/extensions/plugins/RocketsPlugin.h

```diff
@@ -116,12 +118,35 @@ class RocketsPlugin : public ExtensionPlugin
 
     std::unique_ptr<rockets::Server> _rocketsServer;
     using JsonRpcServer = rockets::jsonrpc::Server<rockets::Server>;
-    std::unique_ptr<JsonRpcServer> _jsonrpcServer;
 
     ImageGenerator _imageGenerator;
 
     Timer _timer;
+
+public:
+    std::unique_ptr<JsonRpcServer> _jsonrpcServer;
```

Comment:
style:
- please keep a single public / protect / private section, in this order
- _ prefix is used by convention for private variables and methods (only)